frontend/index.html

```diff
@@ -26,6 +26,11 @@ <h1 class="c-header--title">Simple Push Demo</h1>
   </header>
 
   <main class="l-main">
+    <p>On iOS and iPadOS, to request permission to receive push notifications,
```

Comment:
NIT: Can you move this below the main UI, largely because it's only important to iOS + Safari

Reply:
This should be displayed *before* the user turns on the switch. Otherwise he just can't see this note unless he added the web app to home screen.

Reply:
Ah gotcha. Let me check this out later today (i.e. actually test it)

Reply:
I'm sorry to do this to you, I did a small rejig of the UI so notes were always visible but it does mean you'll need to rebase your PR.

However, I think this will work a lot better going forward and this change totally makes sense, I got notifications working on desktop but not managed it on iOS yet (currently updating).

Reply:
> you'll need to rebase your PR

Done.